src/components/MegaButton/__tests__/MegaButton.spec.js

```diff
@@ -161,7 +195,107 @@ describe('Megabutton', () => {
       });
 
       const text = getByText(props.text);
-      expect(text).toBeValid();
+      expect(text).not.toThrow;
+    });
+
+  });
+
+  describe('Small text with image', () => {
+
+    const textWithImageProps = {
+      id: 'megabutton',
+      modelValue: '',
+      name: 'Fine print',
+      label: 'Megabutton with text',
+      text: 'This also has an image',
+      imageSource: 'fake_image.egg',
+      value: 'bigOlButton'
+    };
+
+    it('renders the label in smaller text', () => {
+      const props = textWithImageProps;
+
+      const { getByText } = render(MegaButton, {
+        props
+      });
+
+      const label = getByText(props.label);
+      expect(label).toHaveClass('text-lg');
+    });
+
+    it('renders the text', () => {
+      const props = textWithImageProps;
+
+      const { getByText } = render(MegaButton, {
+        props
+      });
+
+      const text = getByText(props.text);
+      expect(text).not.toThrow;
```

Comment:
Maybe a better assertion here would be `expect(text).toBeInTheDocument();`

Reply:
Hm, okay, what's the rationale for that?

Reply:
It's just more descriptive of what we're actually testing for.  We want to assert that the element is in the document.  This is instead calling a function that just happens to throw an error if the element was not in the document which is more of a side effect to what we're looking to assert.  Also this depends on using `getBy` versus `queryBy` and people knowing the difference in how they behave, whereas if you assert `toBeInTheDocument` both of those selectors will work interchangeably and require less dependency on the framework.